### DISCUSSION TITLE
Font selection feature

### DISCUSSION THREAD

**Discussion by kajaji** [Stellarium/stellarium#1575] [Q&A] [answered]
The font selection feature in config.ini 
flag_font_selection                 = false
is very useful, but not available by default.

Is there any reason to have it disabled in the default config.ini?

**Comment by alex-w** [comment, marked as answer]
This option for advanced users only, because support regular users, who set "some font" on "some operating system" and now they "see strange text" or "see part text only" will be nightmare if we'll enable this option by default.

**Reply by kajaji** [reply]
Thanks Alex.
